spec/web/authorize_spec.rb

```diff
@@ -99,18 +103,29 @@
 
   context "successful response (swiss post)" do
     before do
-      Datatrans::Web::Transaction::AuthorizeResponse.any_instance.stub(:params).and_return(@successful_swisspost_response)
+      allow_any_instance_of(Datatrans::Web::Transaction::AuthorizeResponse).to receive(:params).and_return(@successful_swisspost_response)
     end
 
     context "process" do
       it "handles a valid datatrans authorize response" do
         @transaction = Datatrans::Web::Transaction.new(@datatrans, @valid_params)
-        @transaction.authorize.should be_true
+        expect(@transaction.authorize).to be true
       end
     end
   end
 
-  context "compromised response" do
+  context 'canceled response' do
```

Comment:
What convention does the gem use? `"` or `'`? Please have it consistent for the gems conventions.